spec/integration/subset_spec.rb

```diff
@@ -36,4 +37,25 @@
     subset1 = TTFunk::Subset.for(font, :unicode)
     expect { subset1.encode }.to_not raise_error
   end
+
+  it 'calculates correct search_range, entry_selector and range_shift values' do
+    font = TTFunk::File.open test_font('DejaVuSans')
+
+    subset = TTFunk::Subset.for(font, :unicode)
+    subset.use(97)
+    subset_io = StringIO.new(subset.encode)
+
+    scaler_type, table_count = subset_io.read(6).unpack('Nn')
+    search_range, entry_selector, range_shift = subset_io.read(6).unpack('nnn')
+
+    expect(scaler_type).to eq(font.directory.scaler_type)
+    # Subset fonts include 13 tables by default.
+    expect(table_count).to eq(13)
+    # Smallest power of two less than number of tables, times 16.
+    expect(search_range).to eq(8 * 16)
+    # Log2 of max power of two smaller than number of tables.
+    expect(entry_selector).to eq(3)
+    # Range shift is defined as 16*table_count - search_range.
+    expect(range_shift).to eq(16 * table_count - search_range)
```

Comment:
`table_count` and `search_range` should be constants in the test.

Reply:
Makes sense, will change.

Reply:
@pointlessone done

Reply:
I'm not sure… It looks the same. 🤷‍♀️

Reply:
Ah sorry, forgot to actually push the update 🤦‍♂️ 

This time it's really done :)